src/components/moviesList/moviesList.component.tsx

```diff
@@ -1,10 +1,14 @@
 import React, { FC } from 'react';
+import { useStoreon } from 'storeon/react';
+import { IMovie } from '@shared/interfaces/movies.model';
+import { StoreModule } from '@app/store/store.interface';
 import MovieItem from '@app/components/movieItem/movieItem.component';
 import IMoviesList from '@app/components/moviesList/moviesList.interface';
 import useStyle from '@app/components/moviesList/moviesList.component.style';
 
-const MoviesList: FC<IMoviesList> = ({ movies, onMovieImageClick }) => {
+const MoviesList: FC<IMoviesList> = ({ onMovieImageClick }) => {
   const s = useStyle();
+  const { movies } = useStoreon(StoreModule.movies);
```

Comment:
```suggestion
  const { movies } = useStoreon<IState>(StoreModule.movies);
```

and no need to assign the `IMovie ` type below

Reply:
Thanks, looks better, constantly forget that there is a possibility to specify interfaces this way.